__tests__/app.test.js

```diff
@@ -91,3 +91,30 @@ describe("Get /api/articles/:article_id", () => {
     expect(body).toEqual({ msg: "Article not found" });
   });
 });
+describe("Get /api/articles", () => {
+  test("200: returns all article objects in an array with the expected properties only", async () => {
+    const { body } = await request(app).get("/api/articles").expect(200);
+    expect(Array.isArray(body)).toBe(true);
+    expect(body.length).toBe(testData.articleData.length);
+    body.forEach(article => {
```

Comment:
if you attempt an array method on an array, do you need an assertion that checks it is an array? Using an array method on anything other than an array will throw an error.